src/clients/shutil.zig

```diff
@@ -219,10 +219,11 @@ pub fn binary_filename(arena: *std.heap.ArenaAllocator, parts: []const []const u
     return file_name.items;
 }
 
-pub fn file_or_directory_exists(arena: *std.heap.ArenaAllocator, f_or_d: []const u8) bool {
-    _ = std.fs.cwd().realpathAlloc(arena.allocator(), f_or_d) catch {
+pub fn file_or_directory_exists(f_or_d: []const u8) bool {
+    var file = std.fs.cwd().openFile(f_or_d, .{}) catch {
         return false;
     };
```

Comment:
Which errors are we ignoring here? Would be good to handle these here explicitly/exhaustively.

Otherwise, if the file does exist, but we encounter an error, we might then do something dangerous.

Reply:
There are quite a few - I've rather made it explicitly `panic` if it encounters something other than `FileNotFound` now.

Reply:
Good call. And now we fail fast at least if we get something unexpected. We only return `false` if it definitely does not exist.